### DISCUSSION TITLE
Deploy in Production Mode (Safe Mode)

### DISCUSSION THREAD

**Discussion by nwarwick** [sidekiq/sidekiq#5981] [Ideas]
Wondering what people think about some kind of production mode for deploying Sidekiq? I had a scare the other day where I was clearing jobs from the dashboard and for a second I thought I was on production.

A production mode could allow for an extra confirmation dialog that appears when trying to complete actions in the dashboard. Or maybe a new set of styles making it clear that you are on production.

Curious what people think!

**Comment by mperham** [comment]
We do put the environment in the browser title. Other suggestions welcome!

**Reply by nwarwick** [reply]
I think maybe if the dashboard is running with a production or safe mode flag a confirmation dialog could display when you try to delete a job or perform a destructive action saying "Sidekiq is running in production mode. Are you sure you would like to delete this job?"

What do you think @mperham?
